crates/hyperqueue/src/client/output/cli.rs

```diff
@@ -363,7 +344,42 @@ impl Output for CliOutput {
                     .unwrap_or("N/A")
                     .cell(),
             ],
+            vec![
+                "Last task started".cell().bold(true),
+                last_task_started
+                    .map(|t| {
+                        format!("Job: {}; Task: {}; Time: {}", t.job_id, t.task_id, t.time).cell()
+                    })
+                    .unwrap_or_else(|| "".cell()),
+            ],
         ];
+        if let Some(runtime_info) = runtime_info {
+            let mut s = String::with_capacity(60);
+            match runtime_info {
+                WorkerRuntimeInfo::SingleNodeTasks {
+                    running_tasks,
+                    assigned_tasks,
+                    is_reserved,
+                } => {
+                    write!(s, "Assigned tasks: {}", assigned_tasks).unwrap();
+                    if running_tasks > 0 {
+                        write!(s, "; Running tasks: {}", running_tasks).unwrap();
+                    }
+                    if is_reserved {
+                        write!(s, "; reserved").unwrap();
```

Comment:
```suggestion
                        write!(s, "; reserved for a multi-node task").unwrap();
```
`reserved` on its own isn't super clear :)